src/test/java/org/opensearch/geospatial/stats/upload/UploadStatsServiceTests.java

```diff
@@ -7,6 +7,7 @@
 
 import static org.opensearch.common.xcontent.XContentFactory.jsonBuilder;
 import static org.opensearch.geospatial.GeospatialTestHelper.buildFieldNameValuePair;
+import static org.opensearch.geospatial.GeospatialTestHelper.removeStartAndEndObject;
```

Comment:
I think this is an unused import. Did we add checkstyle to the repo?

Reply:
This method used to be part of this test, i moved it to Helper class, hence, you see change in import